api_tests/src/wallets/ethereum.ts

```diff
@@ -119,42 +109,51 @@ export class EthereumWallet implements Wallet {
                 this.ethereumDevAccount.address
             );
             const signedTx = await this.ethereumDevAccount.sign(tx);
-            return this.jsonRpcProvider.sendTransaction(signedTx);
+            const transactionResponse = await this.jsonRpcProvider.sendTransaction(
+                signedTx
+            );
+
+            this.logger.debug(
+                "Transaction: ",
+                transactionResponse.hash,
+                " sent, waiting to be confirmed."
+            );
+
+            const transactionReceipt = await this.jsonRpcProvider.waitForTransaction(
+                transactionResponse.hash,
+                1
+            );
+            if (!transactionReceipt.status) {
+                throw new Error(
+                    `Sending amount ${tx.value} to address ${tx.to} failed.`
```

Comment:
Also not entirely true to be honest :see_no_evil: We are using this function to send any kind of TX, also contract deployments which don't send any money. Here is what I would say:

```suggestion
                    `Transaction ${tx.transactionHash} failed with status 1`
```